src/resources/trainers/router.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express")
+
+const router = express.Router();
+
+const { getTrainers } = require("./controller")
```

Comment:
Should probably have this import at the top of the file for the same reason I mentioned in the above comment :)